package.json

```diff
@@ -19,16 +19,17 @@
     "build_closure": "java -jar ./node_modules/google-closure-compiler/compiler.jar ./dist/global/Rx.js --create_source_map ./dist/global/Rx.min.js.map --js_output_file ./dist/global/Rx.min.js",
     "build_global": "rm -rf dist/global && mkdir \"dist/global\" && browserify src/Rx.global.js --outfile dist/global/Rx.js && npm run build_closure",
     "build_perf": "npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
-    "build_test": "rm -rf dist/ && npm run lint && npm run build_cjs && jasmine",
+    "build_test": "rm -rf dist/ && npm run lint && npm run build_cjs && npm run jasmine",
     "build_cover": "rm -rf dist/ && npm run lint && npm run build_cjs && npm run cover",
     "build_docs": "./docgen.sh",
     "lint_perf": "eslint perf/ --fix",
     "lint_spec": "eslint spec/**/*.js --fix",
     "lint_src": "tslint -c .tslintrc src/*.ts src/**/*.ts src/**/**/*.ts",
     "lint": "npm run lint_src && npm run lint_spec && npm run lint_perf",
-    "cover": "istanbul cover -x \"*-spec.js index.js *-helper.js spec/helpers/*\" ./node_modules/jasmine/bin/jasmine.js && npm run cover_remapping",
+    "cover": "istanbul cover -x \"*-spec.js index.js *-helper.js spec/helpers/*\" npm run jasmine && npm run cover_remapping",
```

Comment:
Not sure if I understood question correctly, chaining command via `&&` works with windows command prompt. Problem in here is `istanbul` does not able to consume supplied parameter correctly only on windows, causes trouble.